library/tract-renderer/src/main/java/org/cru/godtools/tract/model/CallToAction.java

```diff
@@ -43,6 +48,9 @@
 
     @ColorInt
     private static int getArrowColor(@Nullable final CallToAction callToAction) {
+        if (callToAction != null && callToAction.mControlColor != null) {
+            return callToAction.mControlColor;
+        }
         return Styles.getPrimaryColor(callToAction != null ? callToAction.getPage() : null);
     }
```

Comment:
I'd split this into 2 functions and rename to `getControlColor` now that we have an actual name for the attribute :)
```
private int getControlColor() {
    return mControlColor != null ? mControlColor : Styles.getPrimaryColor(getPage());
}

private int getControlColor(@Nullable final CallToAction callToAction) {
    return callToAction != null ? callToAction.getControlColor() : Styles.getPrimaryColor(null);
}
```

Reply:
👍